sale_order_min_qty/models/product_min_multiple_mixin.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class ProductMinMultipleMixin(models.AbstractModel):
+    _name = "product.min.multiple.mixin"
+
+    sale_multiple_qty = fields.Float(
+        compute="_compute_sale_min_multiple_qty",
+        store=True,
+        help="Define sale multiple qty"
+        " 'If not set', Odoo will"
+        " use the value defined in the parent object."
+        "Hierarchy is in this order :"
+        "Product/product Template/product category/parent categoroies ",
+        digits=dp.get_precision("Product Unit of Measure"),
+    )
+    manual_sale_multiple_qty = fields.Float(
+        string="Multiple Sale Qty", digits=dp.get_precision("Product Unit of Measure")
+    )
+    sale_min_qty = fields.Float(
+        compute="_compute_sale_min_multiple_qty",
+        store=True,
+        help="Define sale min qty"
+        " 'If not set, Odoo will"
+        " use the value defined in the parent object."
+        "Hierarchy is in this order :"
+        "Product/product Template/product category/parent categoroies ",
```

Comment:
Small remarks, not mandatory:
- there are typo errors and expression to fix 
- there are duplicate strings : if you would use CONSTANT VAR to define strings, there will no duplicates and make it easy to fix
- Missing \n make odoo help popup difficult to read